WebRoot/js/zimbraAdmin/common/ZaApp.js

```diff
@@ -95,6 +95,66 @@ ZaApp.checkMyRight = function(targetType,targetBy,targetVal,right,attrs) {
 	
 }
 
+ZaApp.toggleLoadingStatusBar = function (instance, show, label) {
```

Comment:
instead of introducing a new UX pattern, can we use same pattern that we use for showing a popup when loading large data? by having different UX patterns we are loosing on consistency in product